tests/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@
 # Set the common C++ flags.
 include( detray-compiler-options-cpp )
 include_directories( SYSTEM $<TARGET_PROPERTY:dfelibs::dfelibs,INTERFACE_INCLUDE_DIRECTORIES> )
+include_directories( SYSTEM $<TARGET_PROPERTY:covfie::core,INTERFACE_INCLUDE_DIRECTORIES> )
```

Comment:
Is there a reason to do this over using the target itself, e.g. using `target_link_libraries`?

Reply:
I have no clue, I just copied the line above :D